cnf-certification-test/networking/netcommons/netcommons.go

```diff
@@ -144,3 +146,38 @@ func FilterIPListByIPVersion(ipList []string, aIPVersion IPVersion) []string {
 	}
 	return filteredIPList
 }
+
+func FindRogueContainersDeclaredListeningToPorts(containers []*provider.Container, portsToTest map[int32]bool) []string {
+	var rogueContainers []string
+	for _, cut := range containers {
+		for _, port := range cut.Ports {
+			if portsToTest[port.ContainerPort] {
+				tnf.ClaimFilePrintf("%s has declared a port (%d) that has been reserved", cut, port.ContainerPort)
+				rogueContainers = append(rogueContainers, cut.String())
+				break
```

Comment:
This break will make the check to stop on the first invalid port, but It might be a good idea to report all of them with ClaimFilePrintf():
```
var rogueContainers []string
for _, cut := range containers {
	rogueCut := false
	for _, port := range cut.Ports {
		if portsToTest[port.ContainerPort] {
			tnf.ClaimFilePrintf("%s has declared a port (%d) that has been reserved", cut, port.ContainerPort)
			rogueCut := true
		}
	}
	if rogueCut {
		rogueContainers = append(rogueContainers, cut.String())
	}
}
```

Reply:
Fixed.